AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/README.md

```diff
@@ -54,7 +54,7 @@ This get started sample code is implemented for CPU using the Python language. T
    ```
```

Comment:
Changes required:

## Configure Environment (only for offline installer)

3: Install Jupyter Notebook. (Skip this step if you are working on Intel® DevCloud.)
Suggested change: 3: Install Jupyter Notebook.